src/components/MultiGestureCanvas/types.ts

```diff
@@ -43,8 +43,8 @@ type MultiGestureCanvasVariables = {
     pinchTranslateY: SharedValue<number>;
     stopAnimation: () => void;
     reset: (animated: boolean, callback: () => void) => void;
-    onTap: OnTapCallback | undefined;
-    onScaleChanged?: OnScaleChangedCallback;
+    onTap: OnTapCallback;
```

Comment:
NAB. OnTapCallback is not being exported so we can just define the type directly